src/mcpadapt/core.py

```diff
@@ -101,8 +101,19 @@ async def mcptools(
             f"Invalid serverparams, expected StdioServerParameters or dict found `{type(serverparams)}`"
         )
 
+    timeout = None
+    if client_session_timeout_seconds is not None:
+        if isinstance(client_session_timeout_seconds, float):
+            timeout = timedelta(seconds=client_session_timeout_seconds)
+        elif isinstance(client_session_timeout_seconds, timedelta):
+            timeout = client_session_timeout_seconds
```

Comment:
```suggestion
    if isinstance(client_session_timeout_seconds, float):
        timeout = timedelta(seconds=client_session_timeout_seconds)
    elif isinstance(client_session_timeout_seconds, timedelta):
        timeout = client_session_timeout_seconds
```

sorry last tiny change but I don't think we need the None check as it cannot be instance of float or of timedelta if its None